src/ol/PluggableMap.js

```diff
@@ -61,7 +61,7 @@ import {create as createTransform, apply as applyTransform} from './transform.js
 
 /**
  * @typedef {Object} AtPixelOptions
- * @property {((function(module:ol/layer/Layer): boolean)|undefined)} layerFilter Layer filter
+ * @property {function(module:ol/layer/Layer): boolean|undefined} layerFilter Layer filter
```

Comment:
This makes the type ambiguous.  The type could be a function or undefined.  As it is written here, the function could return a boolean or undefined.

Reply:
Right. In this case you‘d have to put `undefined` first, and then the function.